rfcs/packaging/backup.md

```diff
@@ -0,0 +1,158 @@
+# RFC: Communicating new package structure for `@fluentui/*`
```

Comment:
is this file meant to be a part of the changelist ?

Reply:
nope... 🤦‍♀️ I'll get rid of it.